### PR TITLE
Improve readability

### PR DESCRIPTION
**Pull request by zolaemil**
I think this reads a little better...

### CODE REVIEW

File: lib/dep_graph.js [hunk 1]
```diff
@@ -15,20 +15,20 @@
...
   var visited = {};
-  return function DFS(name) {
-    visited[name] = true;
-    currentPath.push(name);
-    edges[name].forEach(function (edgeName) {
-      if (!visited[edgeName]) {
-        DFS(edgeName);
-      } else if (currentPath.indexOf(edgeName) >= 0) {
-        currentPath.push(edgeName);
+  return function DFS(currentNode) {
```
Comment (jriecken) [inline lib/dep_graph.js]: The changes in this method look ok

File: lib/dep_graph.js [hunk 2]
```diff
@@ -38,33 +38,33 @@ function createDFS(edges, leavesOnly, result) {
  */
 var DepGraph = exports.DepGraph = function DepGraph() {
   this.nodes = {};
-  this.outgoingEdges = {}; // Node name -> [Dependency Node name]
-  this.incomingEdges = {}; // Node name -> [Dependant Node name]
+  this.outgoingEdges = {}; // Node -> [Dependency Node]
+  this.incomingEdges = {}; // Node -> [Dependant Node]
 };
 DepGraph.prototype = {
```
Comment (jriecken) [inline lib/dep_graph.js]: The rest of these changes don't really make sense

E.g. `If a node with the specified node already exists`

This function is adding a node with the specified name.

Comment (zolaemil) [inline lib/dep_graph.js]: Yes, it's just a question of terminology. Take whatever you like of my suggestions! In this implementation a node is nothing else than its name, hence my suggestions.

**Comment by jriecken** [comment]
Looks good with latest changes - thanks!

**Pull request merged**